tests/callbacks/test_EnqueuedEventTimer.cpp

```diff
@@ -49,17 +50,55 @@ TEST_F(EnqueuedEventTimerTest, duration)
 
     this->timer.stop(EndParallelForEvent{.event_id = 1});
 
-    const double elapsed = timer.template duration<Kokkos::utils::timer::milliseconds>().count();
+    const double elapsed = this->timer.template duration<Kokkos::utils::timer::milliseconds>().count();
 
-    //! Check that the elapsed time is greater than the waiting time of 100 ms.
-    ASSERT_GE(elapsed, 100.);
+    //! Check that the elapsed time is greater than 0.
+    ASSERT_GE(elapsed, 0.);
```

Comment:
Probably fine to reduce the `sleep_for` to 1µs ?

Reply:
Or get rid of it ?

Reply:
Introduced an external timer on device after all :)